code/src/components/Loader.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const LoaderImg = styled.img`
+margin-top: 20px;
+`
+const Loader = () => {
+  return (
+    <LoaderImg src="./icons/hero2.gif" alt="hero gif"/>
+  )
```

Comment:
I had no idea you could add a gif like this, nice to see!

Reply:
Super nice loading animation/gif!